templates/land-see/land-see.amp.json

```diff
@@ -0,0 +1,104 @@
+{
+  "land-see": {
+    "head-tag": {
+      "title": "Land + See",
+      "canonical-path": "templates/land-see/land-see.amp",
+      "extensions": [
+        "amp-carousel",
+        "amp-sidebar",
+        "amp-accordion",
+        "amp-instagram",
+        "amp-social-share"
+      ],
+      "css-path": "templates/land-see/page.css",
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Inconsolata|Montserrat:400,500,600,700|Crimson+Text"
+      ]
+    }
+  },
+  "navbar": {
+    "sidebar-id": "header-sidebar",
+    "logo": {
+      "name": "Land + See"
+    },
+    "nav": [
+      {
+        "link": {"url": "stories.amp.html", "nav-item-classes": "sidebar-nav-item", "text": "Stories"},
+        "dropdown": {
+          "initial-state": "expanded",
+          "heading": "Stories",
+          "options": [
+            {
+              "link": {"url": "#", "text": "All"}
+            },
+            {
+              "link": {"url": "#", "text": "Design"}
+            },
+            {
+              "link": {"url": "#", "text": "Fashion"}
+            },
+            {
+              "link": {"url": "#", "text": "Interiors"}
+            },
+            {
+              "link": {"url": "#", "text": "Arts & Culture"}
+            },
+            {
+              "link": {"url": "#", "text": "Popular"}
+            }
+          ]
+        }
+      },
+      {
+        "link": {"url": "#", "nav-item-classes": "sidebar-nav-item", "text": "Shop"}
+      },
+      {
+        "link": {"url": "#", "nav-item-classes": "sidebar-nav-item", "text": "About"}
+      },
+      {
+        "link": {"url": "#", "nav-item-classes": "sidebar-nav-item", "text": "Contact"}
+      }
+    ],
+    "social-follow": {
+      "links": [
+        {"twitter": "#"},
+        {"facebook": "#"},
+        {"instagram": "#"},
+        {"pinterest": "#"},
+        {"email": "#"}
+      ]
+    },
+    "fixeditems": {
+      "link": {
+        "url": "#",
+        "svg": "<svg xmlns='http://www.w3.org/2000/svg' width='11' height='11' viewBox='0 0 11 11' overflow='visible'><circle fill='none' stroke='#000' stroke-width='1.68' stroke-miterlimit='10' cx='4.67' cy='4.67' r='3.83'/><path fill='none' stroke='#000' stroke-width='1.78' stroke-linecap='round' stroke-miterlimit='10' d='M7.258 7.77l2.485 2.485'/></svg>"
+      }
+    }
+  },
+  "footer": {
+    "spacing-classes": "mx-auto max-width-3",
+    "faq-links": {
+      "links": [
+        {"url": "#", "text": "Shop"},
+        {"url": "#", "text": "About"},
+        {"url": "#", "text": "Contact"}
+      ]
+    },
+    "copyright": {
+      "text": "© Land + See, 2017 • ",
+      "terms":
```

Comment:
```json
"tos": {
  "classes": "",
  "link": {
     "url":"#", "text": "text"
   }
}
```
This will line it up close to existing stuff and make it extensible